packetbeat/docs/running-on-docker.asciidoc

```diff
@@ -0,0 +1,25 @@
+include::../../libbeat/docs/shared-docker.asciidoc[]
+
+=== Required Network Capabilities
+
+Under Docker, Packetbeat runs as a non-root user, but requires some privileged
+network capabilities to operate correctly. Ensure that the +NET_ADMIN+
+capability is available to the container.
+
+["source","sh",subs="attributes"]
+----
+docker run --cap-add=NET_ADMIN {dockerimage}
+----
+
+=== Capturing Traffic from the Host System
+
+By default, Docker networking will connect the Packetbeat container to an
+isolated virtual network, with a limited view of network traffic. You may wish
+to connect the container directly to the host network in order to see traffic
+destined for, and originating from, the host system. With +docker run+, this can
+be achieved by specifying +--network=host+.
```

Comment:
Same comment here, for users on Docker for Mac+Windows have a VM layer that won't let them sniff things happening on the real host OS, unless settings have been tweaker.